lib/macros.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#ifdef _WIN32
+#include <windows.h>
+#define OS_HANDLE HMODULE
+#define fptr FARPROC
+#define lib_find(handle,symbol) GetProcAddress(handle,symbol)
+#define drilcp while(!copyFile(GetPath(),CopyPath())){ Sleep(10);}
```

Comment:
I dont feel it is required to make it as a macro. In the interface itself, make a _WIN32 check and call the copy file from there.